docs/source/developer-docs-new/jupyter_enhancement_proposals.rst

```diff
@@ -0,0 +1,30 @@
+Jupyter Enhancement Proposals
+=============================
+
+Submitting an Enhancement Proposal
+----------------------------------
+While using the Notebook, you might discover opportunities for growth and ideas
+for useful new features. If so, feel free to submit an enhancement proposal.
+The process for submitting enhancements is as follows:
+
+1. Identify the scope of the enhancement. Is it a change that affects only on
+   part of the codebase? Is the enhancement, to the best of your knowledge, fairly
+   trivial to implement? If the scope of the enhancement is small, it should be
+   be submitted as an issue in the project's repository. If the scope of your
+   enhancement is large, it should be submitted to the official `Jupyter
+   Enhancement Proposals repository <https://GitHub.com/jupyter/enhancement-proposals>`_.
```

Comment:
This is info that came directly from what already was there. I'm happy to flag and change when I do a section by section editorial review.